lib/entities/entities.js

```diff
@@ -16,26 +17,28 @@ const { EVENTS } = require("../api_definitions");
 class Entities extends EventEmitter {
   #storage;
   #dataPath;
+  #configFileName;
 
   constructor(id) {
     super();
 
     this.id = id;
     this.#storage = {};
-    this.#dataPath = `${this.id}.json`;
+    this.#dataPath = process.env.UC_CONFIG_HOME;
```

Comment:
This breaks all integrations and examples which don't have the environment variable defined!
This env variable needs to be optional and is only present on the remote device.

simulated-light example:
```
node:internal/validators:163
    throw new ERR_INVALID_ARG_TYPE(name, 'string', value);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at new NodeError (node:internal/errors:399:5)
    at validateString (node:internal/validators:163:11)
    at Object.join (node:path:1176:7)
    at new Entities (integration-node-library/lib/entities/entities.js:28:33)
    at new IntegrationAPI (integration-node-library/index.js:39:30)
    at Object.<anonymous> (integration-node-library/index.js:622:18)
    at Module._compile (node:internal/modules/cjs/loader:1246:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1300:10)
    at Module.load (node:internal/modules/cjs/loader:1103:32)
    at Module._load (node:internal/modules/cjs/loader:942:12) {
  code: 'ERR_INVALID_ARG_TYPE'
}
```